doc/develop/microscope.rst

```diff
@@ -91,6 +91,38 @@ Detectors:
 System:
  * lens: Contains parameters concerning the parabolic mirror and the lens system.
 
+Roles of components found in the CRYO/ENZEL system:
+-----------------------------------------------------
+Emitters:
+ * e-beam: Electron beam of the SEM to scan the sample.
+ * light: Controls the excitation light of the FM.
+
+Actuator:
+ * ebeam-focus: Changes the focus position of the e-beam. It has one axis: z. 
+ * filter: Emission filter to select a specific wavelength band.
+ * stage: Moves the sample. It has 3 linear axes (x, y, z) and 2 rotational axes (rx, rz). stage component has the following metadata:
+    1. FAV_POS_DEACTIVE.
+    2. FAV_POS_ACTIVE.
+    3. FAV_POS_COATING.
+    4. FAV_POS_AREA.
+    5. ION_BEAM_TO_SAMPLE_ANGLE.
+ * focus: Changes the distance between the sample and the optical detectors. It has one axis: z. It has two metadata:
+    1. FAV_POS_DEACTIVE.
+    2. FAV_POS_ACTIVE. 
+ * align: Alignment actuator. It has 2 axes: x and y. It has also 2 metadata:
+    1. FAV_POS_ACTIVE.
+    2. FAV_POS_DEACTIVE.
+
+Detectors:
+ * Sample thermostat: Controls the temperature of the sample.
+ * Cooler controller: Controls the starting and stopping of the cooling process by changing the temperature setpoint of the cryo-stage.
+ * se-detector: Secondary electron detector of the SEM. 
+ * ccd: The main optical pixelated detector.
+
+System:
+ * lens: Contains the parameters of the parabolic mirror and the lens system.
```

Comment:
That's a copy of the SECOM system... but the definition is wrong. Let's correct it now. I'd suggest:
`lens: defines the optical parameters (eg, magnification) of the *entire* optical path.`

Please also update the SECOM definition.